astra-sim/system/Sys.cc

```diff
@@ -766,7 +791,7 @@ DataSet* Sys::generate_collective(
   uint64_t chunk_size = determine_chunk_size(size, collective_type);
   uint64_t recommended_chunk_size = chunk_size;
   int streams = ceil(((double)size) / chunk_size);
-  uint64_t remain_size;
+  int tmp;
```

Comment:
Hi, @rashidi1saeed. 

For this one, I think it is because you are not using the latest commit, and `int tmp` is the variable name before changing.

It was modified by me in PR #94 to solve an overflow bugfix and increase readibility. I do not know if "remain_size" is a proper name, but at least keep it type of uint64_t (or int64_t if you need sign, but I guess it do not), otherwise there will be overflow bug.